service/history/queues/dlq_writer.go

```diff
@@ -99,5 +109,17 @@ func (q *DLQWriter) WriteTaskToDLQ(ctx context.Context, sourceCluster, targetClu
 	if err != nil {
 		return fmt.Errorf("%w: %v", ErrSendTaskToDLQ, err)
 	}
+	q.metricsHandler.Counter(metrics.DLQWrites.GetMetricName()).Record(1)
+	ns, err := q.namespaceRegistry.GetNamespaceByID(namespace.ID(task.GetNamespaceID()))
+	if err != nil {
+		q.logger.Warn("Failed to get namespace name", tag.WorkflowNamespace(task.GetNamespaceID()))
+		return nil
+	}
+	q.logger.Warn("Task enqueued to DLQ",
```

Comment:
How often do tasks go to a DLQ? Is this a rare enough occurrence that its worth logging every one of them?

Reply:
We expect it to be a rare event. We should get an alert every time a task is DLQed.